py_zipkin/logging_helper.py

```diff
@@ -332,16 +328,25 @@ def __init__(self, transport_handler, max_portion_size=None):
         self.transport_handler = transport_handler
         self.max_portion_size = max_portion_size or self.MAX_PORTION_SIZE
 
+        if getattr(self.transport_handler, 'get_max_payload_bytes', None):
+            self.max_payload_bytes = self.transport_handler.get_max_payload_bytes()
```

Comment:
Why not have `max_payload_bytes` as an attribute instead of using a function?

Reply:
Because we often end up wanting to reload these values at runtime, so if it's already a function it simplifies our life.

Plus I don't see any particular advantage in having it as an attribute apart from saving 1 line of code.